designer/client/file-upload-field-edit.tsx

```diff
@@ -5,10 +5,15 @@ import { Actions } from "./reducers/component/types";
 import { CssClasses } from "./components/CssClasses";
 import { i18n } from "./i18n";
 
+const defaultOptions = {
+  multiple: false,
+  imageQualityPlayback: false,
+};
+
 export function FileUploadFieldEdit() {
   const { state, dispatch } = useContext(ComponentContext);
   const { selectedComponent } = state;
-  const { options = {} } = selectedComponent;
+  const { options = defaultOptions } = selectedComponent;
```

Comment:
I'd do 

```
const options = { ...defaultOptions, ...selectedComponent.options }
```

e.g. if the default options were (truthy values is a better example)
 
```
{
   multiple: true,
   imageQualityPlayback: true,
}
```

and someone configured only
 
```
{
  multiple: false
}
```

you'd expect the options to be

```
{
   multiple: false,
   imageQualityPlayback: true,
}
```


but since options are no longer undefined, `const { options = defaultOptions } = selectedComponent`, `= defaultOptions` won't be applied, and `imageQualityPlayback` will evaluate to falsy in this case.


It's fine _in this case_ since the defaults are both falsy, but when you have more options or options with a mixture of defined/truthy values this wont work as intended.